<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Customer</class>
 <widget class="QMainWindow" name="Customer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>480</width>
      <height>720</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>واریز وجه</string>
     </attribute>
     <widget class="QTextEdit" name="text_deposit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>4</y>
        <width>441</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>520</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>شماره حساب مورد نظر را وارد کنید</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>560</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>مبلغ واریزی را وارد کنید</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="acc_num_deposit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>520</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="mablagh_deposit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>560</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="deposit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>600</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>تایید</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>برداشت وجه</string>
     </attribute>
     <widget class="QTextEdit" name="text_withdraw">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>441</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>510</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>شماره حساب مورد نظر را وارد کنید</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mablagh_withdraw">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>550</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="acc_num_withdraw">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>510</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>550</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>مبلغ برداشتی را وارد کنید</string>
      </property>
     </widget>
     <widget class="QPushButton" name="withdraw">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>590</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>تایید</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>نمایش اطلاعات </string>
     </attribute>
     <widget class="QTextEdit" name="text_show">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>14</y>
        <width>441</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="show_acc">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>530</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>530</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>شماره حساب مورد نظر را وارد کنید</string>
      </property>
     </widget>
     <widget class="QPushButton" name="show">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>580</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>تایید</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
